test/groovy/com/sap/piper/ConfigurationLoaderTest.groovy

```diff
@@ -56,4 +57,16 @@ class ConfigurationLoaderTest {
         Map config = ConfigurationLoader.defaultGeneralConfiguration(getScript())
         Assert.assertEquals('develop', config.productiveBranch)
     }
+
+    @Test
+    void testIsFeatureActive(){
```

Comment:
&lt;paranoia&gt; Maybe we should also have a test for ```feature3```where ```feature3``` is explicitly provided with ```false```.

Reply:
Done